data/whitelist.json

```diff
@@ -0,0 +1,270 @@
+[
```

Comment:
I found it easier to transform the whitelist into an array and then map it on runtime this helps consistency rather than playing with key values in an object. Be aware though there is not a writing system in place to store the temporary whitelist that the bot may have during runtime.